### ISSUE TITLE
TypeError: marked is not a function when trying to run yarn build or serve

### ISSUE THREAD

**Issue by jgOhYeah** [nrenner/brouter-web#478]
Hello,
I'm not super experienced with nodejs and yarn, so am wondering is anyone would be kind enough to offer a few suggestions. I am trying to get a development version of brouter running on my laptop and am following the instructions in contributing.md, running `yarn` followed by `yarn serve`. This was working with nodejs v10, but I needed to update it to get topo2json to install correctly. I keep getting errors however when running Ubuntu 20.04, Node js v12.22.10 (also tried v16 with the same results) and am getting this output:

```text
$ yarn build debug
yarn run v1.22.15
$ gulp debug
[00:00:44] Using gulpfile /mnt/7C525F7A525F37DA/Users/Jotham/Documents/GitHub/brouter-web/gulpfile.js
[00:00:44] Starting 'debug'...
[00:00:44] Starting '<anonymous>'...
[00:00:44] Finished '<anonymous>' after 945 μs
[00:00:44] Starting 'default'...
[00:00:44] Starting 'clean'...
[00:00:44] Finished 'clean' after 72 ms
[00:00:44] Starting 'scripts_config'...
[00:00:44] Finished 'scripts_config' after 60 ms
[00:00:44] Starting 'layers_config'...
[00:00:44] Finished 'layers_config' after 17 ms
[00:00:44] Starting 'layers'...
[00:00:44] Finished 'layers' after 254 ms
[00:00:44] Starting 'bump:html'...
[00:00:44] Finished 'bump:html' after 8.54 ms
[00:00:44] Starting 'scripts'...
[00:00:44] Running in Debug mode
[00:00:45] Finished 'scripts' after 691 ms
[00:00:45] Starting 'styles'...
[00:00:47] Finished 'styles' after 2 s
[00:00:47] Starting 'images'...
[00:00:47] Finished 'images' after 125 ms
[00:00:47] Starting 'fonts'...
[00:00:47] Finished 'fonts' after 23 ms
[00:00:47] Starting 'locales'...
[00:00:47] Finished 'locales' after 34 ms
[00:00:47] Starting 'boundaries'...
[00:00:47] Finished 'boundaries' after 16 ms
[00:00:47] Starting 'changelog'...
[00:00:47] 'changelog' errored after 1.43 ms
[00:00:47] TypeError: marked is not a function
    at /mnt/7C525F7A525F37DA/Users/Jotham/Documents/GitHub/brouter-web/gulpfile.js:193:40
    at changelog (/mnt/7C525F7A525F37DA/Users/Jotham/Documents/GitHub/brouter-web/node_modules/undertaker/lib/set-task.js:13:15)
    at bound (domain.js:427:14)
    at runBound (domain.js:440:12)
    at asyncRunner (/mnt/7C525F7A525F37DA/Users/Jotham/Documents/GitHub/brouter-web/node_modules/async-done/index.js:55:18)
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
[00:00:47] 'default' errored after 3.3 s
[00:00:47] 'debug' errored after 3.31 s
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Looking online, a [suggested solution](https://stackoverflow.com/a/70647249) to the `TypeError: marked is not a function` error is to change the code of how marked is imported, but seeing as I can't find any similar issues for brouter currently, I don't think this is the case currently and it is more likely my setup. Apart from reinstalling nodejs a few times (which I've done), I'm not sure what to do next.

Thanks in advance.

**Comment by nrenner** [comment]
Thanks for reporting, should now be fixed by 39c56e4.

I recently merged a `marked` update and tested it specifically because of that breaking change, but probably forgot to actually do a `yarn install` :roll_eyes:

**Comment by jgOhYeah** [comment]
Thanks for looking into and fixing it. Works for me now.
